code/plugins/system/tjtokenlogin/tjtokenlogin.php

```diff
@@ -9,7 +9,16 @@
 
 defined('_JEXEC') or die('Unauthorized Access');
 
-require_once JPATH_SITE . '/components/com_api/vendors/php-jwt/src/JWT.php';
+jimport('joomla.filesystem.file');
+
+$jwtFilePath = JPATH_SITE . '/components/com_api/vendors/php-jwt/src/JWT.php';
+
+if (!JFile::exists($jwtFilePath))
```

Comment:
Is this check really needed ?

Reply:
yes - otherwise system plugin (for JWT token based autologin) shows error in some cases

Reply:
Ok can we use JLoader instead of require_once everywhere ?